rosidl_generator_py/resource/_msg.py.em

```diff
@@ -288,12 +294,15 @@ if isinstance(type_, AbstractNestedType):
 @[  end if]@
 ,  # noqa: E501
 @[end for]@
+        rosidl_parser.definition.BasicType('boolean'),  # noqa: E501
     )
 
-    def __init__(self, **kwargs):
-        assert all('_' + key in self.__slots__ for key in kwargs.keys()), \
-            'Invalid arguments passed to constructor: %s' % \
-            ', '.join(sorted(k for k in kwargs.keys() if '_' + k not in self.__slots__))
+    def __init__(self, check_fields=False, **kwargs):
+        self._check_fields = check_fields
```

Comment:
If you change this to:

```suggestion
        self._check_fields = check_fields or ros_python_check_fields
```

Then you only need to do `self._check_fields` below.

Reply:
Thanks and done.